integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/NextMoveServiceBus.java

```diff
@@ -104,18 +106,24 @@ public void putMessage(ConversationResource resource) throws NextMoveException {
             marshaller.marshal(sbd, os);
 
             String queue = NEXTMOVE_QUEUE_PREFIX + resource.getReceiverId();
-            switch (resource.getServiceIdentifier()) {
-                case DPE_INNSYN:
-                    queue = queue + INNSYN.fullname();
-                    break;
-                case DPE_DATA:
-                    queue = queue + DATA.fullname();
-                    break;
-                case DPE_RECEIPT:
-                    queue = queue + receiptTarget();
-                    break;
-                default:
-                    throw new NextMoveException("ServiceBus has no queue for ServiceIdentifier=" + resource.getServiceIdentifier());
+            if (resource.getCustomProperties() != null &&
+                    resource.getCustomProperties().containsKey(MEETING.fullname()) &&
+                    resource.getCustomProperties().get(MEETING.fullname()).equals("true")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "true" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1132)